index.js

```diff
@@ -50,7 +58,21 @@
 // // => "different"
 // console.log("%c----------", "color: red") 
 
-
+function sameSameButDifferent(num, maybeNum) {
+  if (num === maybeNum) {
+    console.log("same same")
+  }
+  else if (num == maybeNum) {
+    console.log("same same (but different)")
+  }
+  else {
+    console.log("different")
+  }
+}
+console.log(sameSameButDifferent(5, 5)) 
```

Comment:
Good job on the completed deliverables. Please at least make an attempt on all questions for these assessments so we can give feedback.